src/views/components/table/Table.web.js

```diff
@@ -513,12 +485,13 @@ class TableView extends Component {
                   disabled={currentPage + 1 >= this.state.totalPages}
                 >
                   <Box
-                    backgroundColor={currentPage + 1 >= this.state.totalPages ?  '#ddd' : '#5173c6'}
+                    backgroundColor={currentPage + 1 >= this.state.totalPages ?  '#ddd' : '#d16360'}
                     padding={10}
                   >
                     <Text
-                      text={isLoadingNextPage || isSearching ? 'Loading...' : 'Next'}
-                      color="white"
+                      text={isLoadingNextPage || isSearching ? 'Loading...' : '>'}
```

Comment:
Use the ternary operator to choose between a Text component with text="Loading..." and an Icon for the arrow

Reply:
@conorpirie  should i close this pull request ?